spec/features/users/sign_in_spec.rb

```diff
@@ -146,7 +146,9 @@ def warning_content
   end
 
   context 'signed in, session times out, sign back in', js: true do
-    it 'prompts to enter OTP' do
+    xit 'prompts to enter OTP' do
+      pending "temporarily disabled until we figure out why it's intermittently failing"
+
```

Comment:
only one of pending or xit are should be needed think.  see https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples
please reference the bug in a TODO comment.